res/skins/LateNight/style.qss

```diff
@@ -1358,15 +1376,25 @@ QTableView:focus, QTreeView:focus {
   border: 1px solid #FF6600;
 }
 
-/* checkbox in library "Played" column */
-QTableView::indicator { width: 12px; height: 12px;}
-QTableView::indicator:checked { background: url(skin:/style/style_checkbox_checked.png);}
-QTableView::indicator:unchecked { background: url(skin:/style/style_checkbox_unchecked.png);}
+/* selected items in Tree and Tracks table */
+QTreeView::item:selected,
 QTableView::item:selected {
-  color: #cfb32c;
+  color: #fff;
   background-color: #725309;
 }
 
+/* checkbox in library "Played" column */
+QTableView::indicator {
+  width: 12px;
+  height: 12px;
+  }
+  QTableView::indicator:checked {
+    background: url(skin:/style/library_checkbox_checked.png);
+  }
+  QTableView::indicator:unchecked {
+    background: url(skin:/style/library_checkbox_unchecked.png);
+  }
+
 /* BPM lock icon in the library "BPM" column. */
 #LibraryBPMButton::indicator:checked {
   image: url(:/images/library/ic_library_checked.png);
```

Comment:
same for "BPM lock icon":

```
#LibraryBPMButton::item:hover,
#LibraryBPMButton::item:checked:hover {
   border: 1px solid #eeeeee;
}
```
![bpm](https://user-images.githubusercontent.com/3403218/40567395-5a004e38-6075-11e8-8038-8d986b2a3fe4.png)

Reply:
same for : checkbox in library "Played" column 
```
QTableView::indicator:hover,
QTableView::indicator:checked:hover {
   border: 1px solid #eeeeee;
}
```
![played](https://user-images.githubusercontent.com/3403218/40568254-cd15c5b6-6079-11e8-830f-3b6de705bd16.png)